Horizon/Horizon/Sources/Features/Notebook/Common/View/NoteCard.swift

```diff
@@ -0,0 +1,147 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2024-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUICore
+import SwiftUI
+
+private let textDisabledColor = Color(red: 27/100, green: 36/100, blue: 45/100)
+
+struct NoteCard: View {
+    let note: NotebookNote
+
+    init(_ note: NotebookNote) {
+        self.note = note
+    }
+
+    var body: some View {
+        NotebookCard {
+            VStack(alignment: .leading, spacing: 8) {
+                Text(note.title).font(.regular12).padding(.bottom, 8)
+                Text(note.note).font(.regular16).padding(.bottom, 8)
```

Comment:
The preferred way of writing modifiers (and e.g functions with multiple parameters, reactive operators) is having them separated on new lines like this: 

```
                Text(note.title)
                    .font(.regular12)
                    .padding(.bottom, 8)
                Text(note.note)
                    .font(.regular16)
                    .padding(.bottom, 8)
```